configs/accengage.json

```diff
@@ -1,30 +1,41 @@
 {
   "index_name": "accengage",
   "start_urls": [
-    "https://docs-alpha.accengage.com"
+    "https://documentation.accengage.com",
+    "https://documentation.accengage.com/sdk",
+    "https://documentation.accengage.com/api",
+    "https://documentation.accengage.com/dataflow/introduction",
+    "https://documentation.accengage.com/user-guide/en",
+    "https://documentation.accengage.com/user-guide/fr",
+    {
+      "url": "https://documentation.accengage.com/faq",
+      "selectors_key": "faq"
+    }
   ],
-  "stop_urls": [],
+  "stop_urls": ["https://documentation.accengage.com/admin"],
   "selectors": {
-    "lvl0": {
-      "selector": "(//nav//a[contains(@class, 'active')])[1]/../../preceding::a[1]",
-      "type": "xpath",
-      "global": true,
-      "default_value": "Documentation"
+    "default": {
+      "lvl0": "h1, #title-text",
```

Comment:
These selectors are too broad since they can match any element within the page. They should be scope to the to smallest element gathering your whole documentation. This element should be the main container of your textual content. `.main-content` seems to be a good fit.